tests/test_modeling_tf_common.py

```diff
@@ -435,6 +459,8 @@ def test_inputs_embeds(self):
             del inputs_dict["decoder_input_ids"]
 
         for model_class in self.all_model_classes:
+            if model_class in TF_MODEL_FOR_MULTIPLE_CHOICE_MAPPING.values():
```

Comment:
Is that not possible currently?

Reply:
It requires some additional preprocessing + the `inputs_embeds` need to be flattened in the model, I hadn't done #4921 yet so I wasn't sure how hard it would be to add. Will see if I can add it in this PR (and if it takes too long/too much code, will do a separate PR).

Reply:
It wasn't too bad in the end, added it.